tests/bootstrap.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Predis-Toolkit
+ *
+ * Additional classes and functionality to extend Predis
+ *
+ * @copyright 2015 One Mighty Roar
+ * @link http://onemightyroar.com
+ */
+
+// Load our autoloader, and add our Test class namespace
+$autoloader = require(__DIR__ . '/../vendor/autoload.php');
+$autoloader->add('OneMightyRoar\PredisToolki\Tests', __DIR__);
```

Comment:
Instead of doing this manually here, I'd use the newer `autoload_dev` bits in the `composer.json` to setup the autoloader for you: https://getcomposer.org/doc/04-schema.md#autoload-dev